packages/client/components/DiscussionThreadListEmptyState.tsx

```diff
@@ -131,18 +131,18 @@ const DiscussionThreadListEmptyState = (props: Props) => {
       }
     }
   })
+  if (!settings) return null
+  const {teamId, videoMeetingURL} = settings
```

Comment:
-2 I only noticed in #8588 that the meeting settings are not per meeting. We don't prohibit having 2 retros at the same time, thus the settings would be shared. The correct way to handle this would be moving the settings from the meeting settings to the meeting on meeting creation.

Reply:
I've updated this to move the logic to the meeting object and clear `videoMeetingURL` from the settings once the meeting starts.

The PR still works for me, but I'm interested to know if it still works for you as Zoom recently announced that they're changing their policy on bots so we'll need to create an oauth flow:

![Screenshot 2023-09-26 at 16 38 39](https://github.com/ParabolInc/parabol/assets/39854876/7f4b0175-cb3d-48ee-bee5-ba78d7e678ba)